utils/customPropTypes.js

```diff
@@ -85,6 +85,7 @@ export const ArticleModel = {
   tagsByTopic: PropTypes.objectOf(TagsArray).isRequired,
   theme: ThemeType.isRequired,
   color: PropTypes.string.isRequired,
+  metrics: PropTypes.number,
```

Comment:
number?

Reply:
well after I transform a map - it is a single number